### PR TITLE
CAT-1147 erb to epp conversion

### PR DESCRIPTION
**Pull request by dinesh-gujar**
## Summary
Conversion of ERB to EPP templates for mysql module

## Checklist
- [ ] 🟢 Spec tests.
- [ ] 🟢 Acceptance tests.
- [ ] Manually verified. (For example `puppet apply`)

### CODE REVIEW

**Comment by malikparvez** [comment]
can you squash your commits?

File: templates/xtrabackup.sh.epp [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+cleanup()
+{
+    find "${DIR}/" -mindepth 1 -maxdepth 1 -mtime +${ROTATE} -print0 | xargs -0 -r rm -rf
+}
+
+<% if $delete_before_dump { -%>
+cleanup
+<% } -%>
+
```
Comment (Ramesh7) [inline templates/xtrabackup.sh.epp:35]: are these newlines intended ?

Comment (dinesh-gujar) [reply templates/xtrabackup.sh.epp:35]: yes

File: lib/puppet/functions/mysql/innobackupex_args.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
```
Comment (Ramesh7) [inline lib/puppet/functions/mysql/innobackupex_args.rb:2]: can we put some description about this function which will help newbie to onboard faster?

Comment (dinesh-gujar) [reply lib/puppet/functions/mysql/innobackupex_args.rb:2]: yes sure

**Review by Ramesh7 — APPROVED** [review]
lgtm, waiting for CI to get 🟢

**Pull request merged**